<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chioce</class>
 <widget class="QDialog" name="chioce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="sum_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>当前连接人数：0人</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>服务器未开启</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭服务器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查看日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开启服务器</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
